protocol/avdl/chat1/local.avdl

```diff
@@ -36,6 +36,7 @@ protocol local {
     string mimeType;           // mime type of the object
     Hash encHash;              // hash of ciphertext object 
     bytes key;                 // encryption key
```

Comment:
I left it `key` for backwards-compatibility as this struct already used in messages.  To change the name would require a version bump or database wipe.